plugins/@grouparoo/iterable/src/lib/connect.ts

```diff
@@ -0,0 +1,5 @@
+import { SimpleAppOptions } from "@grouparoo/core";
+
+export async function connect(appOptions: SimpleAppOptions) {
+  return require('node-iterable-api').create(appOptions.apiKey);
```

Comment:
Let's `import Iterable from 'node-iterable-api'` (ideally) or `cosnt Iterable = require('node-iterable-api')` (if necessary) above and then do `Iterable.create()` here.